subsys/bluetooth/controller/include/ll.h

```diff
@@ -206,3 +206,38 @@ void ll_rx_mem_release(void **node_rx);
 void ll_timeslice_ticker_id_get(uint8_t * const instance_index, uint8_t * const user_id);
 void ll_radio_state_abort(void);
 uint32_t ll_radio_state_is_idle(void);
+
+/* Direction Finding */
+
+/* @brief Function sets CTE transmission parameters for a connection.
+ *
+ * @param[in]handle			Connection handle.
+ * @param[in]cte_types			Bitfield holding information about
```

Comment:
Tabs used for indentation here, which should be spaces. 
This is why it ends up differently aligned.

Reply:
Tabs changed to spaces